### ISSUE TITLE
Consider using react-style

### ISSUE THREAD

**Issue by nelix** [OlavHN/react-paper#2]
https://github.com/js-next/react-style
Although I do love how simple yet effective css.js is

**Comment by OlavHN** [comment]
Agreed! I tried with a similar approach earlier but got stuck at the silly flex attribute.

All modern browsers require `display: flex`, while safari insists on `display: -webkit-flex`

That causes issues trying to define the flex attribute in a javascript object. Got any workarounds? I tried doing a runtime check in componentWillMount, but that was even uglier :)
